insight/pv_model/pv_model_class.py

```diff
@@ -322,12 +317,12 @@ def calculate_power(self, Rs, Rp, a):
         deltaT = T - Tn
 
         # current equations
-        Ipvn, Ipv, Isc, I0 = self.calculate_currents(
+        Ipvn, Ipv, I0 = self.calculate_currents(
             Rs, Rp, a, Iscn, Vocn, Ki, Kv, Vt, Ns, G, Gn, deltaT
         )
 
-        v = np.arange(0, (Vocn + Kv * deltaT), increment)
-        i = np.ones_like(v) * Iscn
+        v = np.arange(0, Vocn, increment)
+        i = np.zeros_like(v)
```

Comment:
Why did you make this change?

Reply:
It should be fine either way. I don't remember the difference in the bound for the voltage, but I seem to remember us experimenting with the initialization of the current values. Since the majority of the current values are close to the short-circuit current value (which we know due to the shape of the I-V curve), initializing the current at the short circuit current should make it converge faster when calculating through Newton-Raphson.